### PR TITLE
Channels integration

### PR DESCRIPTION
**Pull request by lepatrick714**
Added email dups checks and fixed issue with facebook and google registration

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -43,11 +43,13 @@ def do_admin_login():
...
-        if newUser.validateUser(): 
-            return jsonify({u'auth': u'success'})
+        uid = newUser.getUserUid()
+        if newUser.validateUser() is True: 
+            print("HERE")
+            return jsonify({u'userId': uid})
         else: 
-            return jsonify({u'auth': u'failure'})
+            print("ERROR")
+            return jsonify({u'auth': u'-1'})
```
Comment (ttony678) [inline app.py]: Not sure, but I think we should always send back the same key for the key value pair. If I try to do getString("userId") but an "auth" is sent back we'll get an exception error.

Comment (lepatrick714) [reply app.py]: Fixed!

**Review by ttony678 — CHANGES_REQUESTED** [review]
Can you change the 'auth' to 'userId' too?

**Comment by lepatrick714** [comment]
Fixed

**Review by ttony678 — APPROVED** [review]

**Pull request merged**